app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,13 @@
+@import 'sass/colors';
+
+#error_explanation {
```

Comment:
Avoid using id selectors<br>Selector `error_explanation` should be written in lowercase with hyphens